backend/app/api/ws/local_executor_namespace.py

```
@@ -0,0 +1,703 @@
+        Args:
+            sid: Socket ID
+            data: Result data containing task_id, subtask_id, status, result
+
+        Returns:
+            {"success": true}
+        """
+        task_id = data.get("task_id")
+        subtask_id = data.get("subtask_id")
+        status = data.get("status")
+        result = data.get("result", {})
+        message = data.get("message", "")
```

Ruff (0.14.14)

Local variable `message` is assigned to but never used

Remove assignment to unused variable `message`

(F841)